README.md

```diff
@@ -27,5 +27,5 @@ Until wasm-bindgen releases my changes to crates.io you will have to install was
 cargo install -f wasm-bindgen-cli --git https://github.com/rustwasm/wasm-bindgen.git  --rev 27173549f6a196a869cec525f8f87adec55c738c
```

Comment:
I am using an unreleased version of wasm-bindgen this is why I need to specify the rev